content/applications/sales/sales/amazon_connector/update.rst

```diff
@@ -48,25 +40,19 @@ Install the Amazon/Authentication Patch
 =======================================
 
 .. warning::
-    You should never install new modules in your production database without testing
-    them in a duplicate or staging environment. For Odoo.com customers, a duplicate database
-    can be created from the database management page. For
-    Odoo.sh users, you should use a staging or duplicate database. For on-premise users,
-    you should use a staging environment - you should contact your integrating partner for
-    more information regarding how to test a new module in your particular setup.
-
-The module should now be available in your **Apps** menu. Remove the ``Apps`` search facet
-and search for ``amazon``; the module **Amazon/Authentication Patch** should be available for
-installation. If you cannot find the module after having updated the list of available
-modules, it means your Odoo source code is not up-to-date; refer to step one of this
-page.
-
-.. image:: ./media/auth_module.png
-  :align: center
-
-Once the module is installed, you will need to generate an Authorization Token in Amazon
-Seller Central and set it up on your Amazon Account in Odoo; this process is detailed
-in the :doc:`setup <setup>` page.
+   You should never install new modules in your production database without testing them in a
+   duplicate or staging environment. For Odoo.com customers, a duplicate database can be created
+   from the database management page. For Odoo.sh users, you should use a staging or duplicate
+   database. For on-premise users, you should use a staging environment--contact your integrating
```

Comment:
Let's fix this punctuation either with a semicolon `;` or an em dash `---` as we discussed :)
```suggestion
   database. For on-premise users, you should use a staging environment---contact your integrating
```